specification/cognitiveservices/data-plane/Face/stable/v1.0/Face.json

```diff
@@ -4238,6 +4268,24 @@
       "type": "boolean",
       "in": "query",
       "x-ms-parameter-location": "method"
+    },
+    "detectionModel": {
+      "name": "detectionModel",
+      "description": "The 'detectionModel' associated with the detected faceIds. Supported 'detectionModel' values include \"detection_01\" or \"detection_02\"",
```

Comment:
Not only detected faceIds, but also the persisted faces in PG/LPG/FL/LFL.

Reply:
Following the same style as the above `recognitionModel` should be good.

Reply:
Done =)